Fix azure-mgmt-datalake-store link to docs

wjohnson opened the pull request:
The link for Azure Management for Data Lake Store leads to a 404 page on readthedocs.  I found the correct page and have provided the updated link.

azurecla commented:
Hi __@mahthi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mahthi). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
Merged through private repo

wjohnson committed:
Fix azure-mgmt-datalake-store link to docs

The link for Azure Management for Data Lake Store leads to a 404 page on readthedocs.  I found the correct page and have provided the updated link.
